Src/WitsmlExplorer.Api/HttpHandlers/JobHandler.cs

```diff
@@ -100,5 +100,11 @@ public static IResult CancelJob(string jobId, IJobCache jobCache)
             }
             return TypedResults.NotFound();
         }
+
+        [Produces(typeof(IEnumerable<object>))]
+        public static IResult GetReportItems(string jobId, IJobCache jobCache)
+        {
+            return TypedResults.Ok(jobCache.GetReportItems(jobId));
```

Comment:
We need to make sure that the user has permission to read that report. I think you can do the same as in GetUserJobInfo. I suggest not changing anything in the job cache, and rather use GetUserJobInfo directly, and then return the report part of the jobInfo. Then the permission verification is performed by GetUserJobInfo. We could then add [JsonIgnore] to the Report in JobInfo so that it is excluded when fetching job infos, so that it's only available through the new endpoint.

Reply:
Instead of just returning the ReportItems, I think it makes more sense to have an endpoint that returns the whole report. Then we should also rename it and the endpoint accordingly.

Reply:
fixed